R/cqcp_helper.R

```diff
@@ -83,7 +83,7 @@ cqcp_check_input <- function(data, print = TRUE, file = NULL, as_list = FALSE){
   if(!has_z) {
     miss_1b <- c(miss_1b, "z")
     mess_1b <- cqcp_colourise("     ! Missing: z\n", "yellow")
-    mess_1b <- cqcp_colourise("     --> Filters cqcp_m2 and cqcp_m5 will not work with 'heightCorrection = T'. You can run 'cqcp_add_dem_height' to add DEM information.\n", "yellow")
+    mess_1b <- c(mess_1b, cqcp_colourise("     --> Filters cqcp_m2 and cqcp_m5 will not work with 'heightCorrection = T'. You can run 'cqcp_add_dem_height' to add DEM information.\n", "yellow"))
```

Comment:
is `mess_1b` intentional? or should it be `miss_1b`?

Reply:
Yes, intentional.
`mess_1b` for message, `miss_1b` for what is missing.